src/App.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import Todo from './Todo';
 
 // rework this into regular api call, feel free to use any open api
-var todos = (): Promise<{id: string; title: string;}[]> => new Promise((res) => {
```

Comment:
Naming convention. It's good to know that that is a service, and it's getting data from API.
Use const instead of var.